iOS/Source/MainView.swift

```diff
@@ -201,8 +201,8 @@ class MainView: UIView {
 
             UIView.animate(withDuration: 0.4) {
                 self.backgroundColor = backgroundColor
-                self.sunView.updateInterface(withBackgroundColor: backgroundColor, textColor: textColor, andPercentageInDay: percentageInDay, sunPhase: sunPhase)
                 self.sunView.update(for: location)
+                self.sunView.updateInterface(withBackgroundColor: backgroundColor, textColor: textColor, andPercentageInDay: percentageInDay, sunPhase: sunPhase)
```

Comment:
Why did we need to move this method bellow? If this an issue of one overwriting the other, can we merge both methods into one?

Can we merge this two:

<img width="273" alt="screen shot 2017-01-23 at 12 03 26 pm" src="https://cloud.githubusercontent.com/assets/1088217/22201450/f811a5b4-e163-11e6-81e0-44f2b961dfa0.png">